core/src/main/java/org/apache/iceberg/BaseMetastoreCatalog.java

```diff
@@ -143,6 +145,31 @@ public Table loadTable(TableIdentifier identifier) {
     }
   }
 
+  public boolean createNamespace(Namespace namespace) {
+    return false;
+  }
+
+  public List<Namespace> listNamespaces() {
+    return null;
+  }
+
+  public List<Namespace> listNamespaces(Namespace namespace) {
+    return null;
+  }
+
+  public Map<String, String>  loadNamespaceMetadata(Namespace namespace) {
+    return null;
+  }
+
+  public boolean dropNamespace(Namespace namespace) {
+    return false;
+  }
+
+  public boolean alterNamespace(Namespace namespace) {
+    return false;
+  }
```

Comment:
I dont think we need to implement these dummy methods in base class, anyway we have to implement them per catalog.

Reply:
fixed:change it to BaseMetastoreCatalog